src/main/java/org/fundacion/jala/converter/models/facade/ParameterOutputChecksum.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Gustavo Zacarias Huanca Alconz
+ */
+package org.fundacion.jala.converter.models.facade;
+
+public class ParameterOutputChecksum {
+    private String checksumLocal;
+    private String outputFilename;
+    private int resultTitleSize;
+    private String filename;
+
+    public ParameterOutputChecksum(final String checksumLocal, final String outputFilename, final int resultTitleSize, String filename) {
```

Comment:
The parameter filename should be final

Reply:
The lines of code should not exceed 120 characters

Reply:
ok